apps/sms/views/conversation/js/conversation.js

```diff
@@ -3015,6 +3022,37 @@ var ConversationView = {
     threadMessagesClass.remove('editable-select-mode');
     node.focus();
     window.getSelection().selectAllChildren(node);
+  },
+
+  initLowStorageHandling: function() {
```

Comment:
There functions are reused between 2 views, maybe I  should do a device storage mixin for view(or even replace the device storage client/service)?